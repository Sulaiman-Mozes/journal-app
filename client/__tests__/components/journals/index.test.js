import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import JournalItems from '../../../components/journals';

describe('Provider and JournalItems', () => {
  const props = {
    handleDelete: jest.fn(),
    notes: [{ _id: 'q26738wueyihg98', title: 'title', content: 'content' }],
  };

  const journalItems = shallow(<JournalItems {...props} />);

  it('renders <Provider/> correctly', () => {
    expect(journalItems).toMatchSnapshot();
  });
});
