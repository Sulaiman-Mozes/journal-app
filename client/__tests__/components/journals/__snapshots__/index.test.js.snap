// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Provider and JournalItems renders <Provider/> correctly 1`] = `
<div
  className="container align-center"
>
  <div
    className="card-columns"
  >
    <div
      className="card mb-4"
      key="q26738wueyihg98"
    >
      <div
        className="card-body"
      >
        <h5
          className="card-title"
        >
          title
        </h5>
        <p
          className="card-text"
        >
          content
        </p>
        <p
          className="card-text"
        >
          <small
            className="text-muted"
          >
            Last updated
             
            0 minute(s)
             
            ago
          </small>
        </p>
        <div
          className="d-flex justify-content-around"
        >
          <NavLink
            to="/update/q26738wueyihg98"
          >
            <button
              className="btn btn-primary btn-sm"
              type="button"
            >
              Update
            </button>
          </NavLink>
          <button
            className="btn btn-danger btn-sm"
            data-target="#modalConfirmDelete"
            data-toggle="modal"
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
